catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -91,25 +103,39 @@ define([
             }
 
             if(!this.get('multiValued')) {
-                if(this.get('type') === 'number'){
+                if(type === 'number'){
```

Comment:
This is also more of a "for future consideration" comment, but whenever I see objects with a lot of conditional logic based on some kind of `type` field, I think that it's time to consider breaking that object into subtypes instead. So you might have a `NumberField` and a `StringField` and so on that are subtypes of an `AbstractField` that contains the common logic. It eliminates a lot of branches and makes it easier to read and test.